doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -78,15 +80,15 @@ of the device), then the :ref:`ceph-volume-lvm` call for an lv named
 
 **block and block.db**
 ^^^^^^^^^^^^^^^^^^^^^^
-If there is a mix of fast and slow devices (spinning and solid state),
+If you have a mix of fast and slow devices (SSD / NVMe and rotational),
 it is recommended to place ``block.db`` on the faster device while ``block``
-(data) lives on the slower (spinning drive). Sizing for ``block.db`` should be
-as large as possible to avoid performance penalties otherwise. The
```

Comment:
why are we dropping this line?

Reply:
It conflicts with the sizing advice below.  I thought it best to consolidate sizing advice in one place.